### PR TITLE
inadyn 2.5 (new formula)

### PR DESCRIPTION
**Pull request by jorhett**
Implemented per discussion in #32346

### CODE REVIEW

**Comment by jorhett** [comment]
I've read the Jenkins logs and it's not clear to me why it is failing. Seems to be around the `brew doctor` command, which has nothing to do with this change, no?

**Comment by fxcoudert** [comment]
@jorhett that `brew doctor` error is because our CI machines need a CLT update, please disregard.

However, there is a build failure on Sierra:

```
../lib/utimensat.c:40:15: error: use of undeclared identifier 'AT_SYMLINK_NOFOLLOW'
        if ((flags & AT_SYMLINK_NOFOLLOW) == AT_SYMLINK_NOFOLLOW)
                     ^
../lib/utimensat.c:40:39: error: use of undeclared identifier 'AT_SYMLINK_NOFOLLOW'
        if ((flags & AT_SYMLINK_NOFOLLOW) == AT_SYMLINK_NOFOLLOW)
                                             ^
2 errors generated.
```

**Comment by jorhett** [comment]
I have no machines left on Sierra. It works fine on High Sierra.

@troglobit It would appear that this is a common problem where we need an ifdef for older macOS, I think it's well described here: mpartel/bindfs#6

**Comment by jorhett** [comment]
@fxcoudert just to be clear-- this builds successfully on High Sierra and Mojave, this only affects Sierra?

**Comment by fxcoudert** [comment]
@jorhett indeed

**Comment by troglobit** [comment]
Sierra build fix available upstream in troglobit/inadyn@57bdcc0. Possibly there is a way to bundle patches in Homebrew, e.g. like Debian does. But I have no idea since I don't use macOS myself.

File: Formula/inadyn.rb
Comment (vszakats) [inline Formula/inadyn.rb]: This redirects to <http://troglobit.com/projects/inadyn/>. Would you mind updating?

**Comment by jorhett** [comment]
@troglobit I've updated the Formula with the patch. It works on High Sierra

```
==> Patching
==> Applying 57bdcc0321b49ee68397c70140d9895655edb06f.patch
patching file lib/pidfile.c
patching file lib/utimensat.c
```

@vszakats I've updated the URL as you requested.

File: Formula/inadyn.rb
Comment (fxcoudert) [inline Formula/inadyn.rb]: Please use `https://github.com/troglobit/inadyn/commit/57bdcc03.diff?full_index=1` as URL (and adapt the checksum below). It is more stable over time.

File: Formula/inadyn.rb
Comment (fxcoudert) [inline Formula/inadyn.rb]: We do not want `-W portability` warnings, please remove.

File: Formula/inadyn.rb
Comment (fxcoudert) [inline Formula/inadyn.rb]: The caveat is generic advice about using the software, and not Homebrew specific I guess. If so, please remove it.

**Review by fxcoudert — CHANGES_REQUESTED** [review]
Thanks for the changes, we're almost there!

File: Formula/inadyn.rb
Comment (fxcoudert) [reply Formula/inadyn.rb]: `  * C: 12: col 3: `depends_on` (line 12) should be put before `patch` (line 7)`

File: Formula/inadyn.rb
Comment (jorhett) [reply Formula/inadyn.rb]: The paths are homebrew specific, in particular where to find the template files. I'm happy to rephrase or approach this however you want, but expecting the user to grep around the filesystem or use `brew list inadyn` to find the config files seems very unfriendly. A great many formula provide instructions on how to get started.

Again, happy to provide in whatever manner you feel is best.

**Comment by jorhett** [comment]
Except the `caveots` where I'm waiting for a reply, all other changes requested have been completed in this latest push @fxcoudert

File: Formula/inadyn.rb
Comment (commitay) [reply Formula/inadyn.rb]: They are standard paths that are commonly used within Homebrew. The caveat is unnecessary, please remove it.

**Review by commitay — DISMISSED** [review]

File: Formula/inadyn.rb
Comment (fxcoudert) [inline Formula/inadyn.rb]:
```suggestion
  # Fix Sierra build, remove in next version
  patch do
```

Comment (jorhett) [reply Formula/inadyn.rb]: Everything you're asking for is reasonable, but can we avoid having a dozen separate requests for one-line changes?  Why not give them all to me at once?

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
@jorhett because we do what we can, you had feedback from different maintainers…

**Pull request closed without merging**
